core/src/main/java/io/cloudevents/core/builder/CloudEventBuilder.java

```diff
@@ -279,4 +281,22 @@ static CloudEventBuilder from(@Nonnull CloudEvent event) {
         );
     }
 
+    /**
+     * Create a new builder starting from the values of the provided context.
+     *
+     * @param context context to copy values from
+     * @return the new builder
+     */
+    static CloudEventBuilder fromContext(@Nonnull CloudEventContext context) {
+        switch (context.getSpecVersion()) {
+            case V1:
+            return new io.cloudevents.core.v1.CloudEventBuilder(context);
+            case V03:
+            return new io.cloudevents.core.v03.CloudEventBuilder(context);
+        }
+        throw new IllegalStateException(
+            "The provided spec version doesn't exist. Please make sure your io.cloudevents deps versions are aligned."
+        );
+    }
+
```

Comment:
This method is unused, can we remove it?

Reply:
It's not unused, it's a new api

Reply:
Ok, cool!